.husky/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+. "$(dirname -- "$0")/_/husky.sh"
+
+# Only save the files that were stagged for the commit
+stagged_files=$(git diff --name-only --cached --diff-filter=ACMRTUB)
+
+# formatting
+mix format
+npm run --prefix apps/site/assets prettier -- -staged
+
+# linting
+npm run --prefix apps/site/assets eslint:js
+npm run --prefix apps/site/assets eslint:ts
```

Comment:
```suggestion
npm run ci:lint:js
npm run ci:lint:ts
```
suggestion: There are existing scripts for these in the root `package.json` file.